sycl/test/basic_tests/interop-hip.cpp

```diff
@@ -51,7 +51,7 @@ int main() {
   // backend-defined and specified in the backend specification.
 
   hip_device = get_native<backend::ext_oneapi_hip>(Device);
-  // expected-warning@+1{{'get_native<sycl::backend::ext_oneapi_hip, sycl::context>' is deprecated: Context interop is deprecated for HIP. If a native context is required, use hipDevicePrimaryCtxRetain with a native device}}
+  // expected-no-diagnostics
   hip_context = get_native<backend::ext_oneapi_hip>(Context);
```

Comment:
I'm curious about why don't we run this LIT test? Since call to `get_native<backend::ext_oneapi_hip>(Context)` should now be throwing an exception instead of a depreciation warning and to check the exception in LIT test, don't we have to run the test?